crates/astria-composer/src/collectors/geth.rs

```diff
@@ -64,17 +64,17 @@ const WSS_UNSUBSCRIBE_TIMEOUT: Duration = Duration::from_secs(2);
 pub(crate) struct Geth {
     // Chain ID to identify in the astria sequencer block which rollup a serialized sequencer
     // action belongs to. Created from `chain_name`.
-    rollup_id: RollupId,
+    pub(super) rollup_id: RollupId,
```

Comment:
We had to increase the visibility for the GethCollectorBuilder.

Reply:
Undo this. The point of making these builders is to keep the implementation details of these objects private.

Reply:
In the `collectors` module, we can have geth and grpc as seperate modules.